### DISCUSSION TITLE
Question about Content-Security-Policy

### DISCUSSION THREAD

**Discussion by yanghuaxuan** [misskey-dev/misskey#11362] [Q&A] [answered]
Hi, I was wondering how [misskey.io](https://misskey.io) is able to deploy Misskey without adding "script-src 'unsafe-inline'" to the Content-Security-Policy header.

**Comment by acid-chicken** [comment, marked as answer]
The [misskey.io](https://misskey.io) made changes regarding CSP in their own fork (not [misskey-dev/misskey](https://github.com/misskey-dev/misskey)). https://github.com/misskey-dev/misskey/compare/develop...MisskeyIO:misskey:io
